src/elements/content-sidebar/versions/VersionsSidebar.js

```diff
@@ -12,24 +12,36 @@ import messages from './messages';
 import SidebarContent from '../SidebarContent';
 import VersionsMenu from './VersionsMenu';
 import { BackButton } from '../../common/nav-button';
+import PlainButton from '../../../components/plain-button/PlainButton';
 import { DEFAULT_FETCH_END } from '../../../constants';
 import { LoadingIndicatorWrapper } from '../../../components/loading-indicator';
 import type { BoxItemVersion } from '../../../common/types/core';
+import type { openUpgradePlanModal } from './flowTypes';
 import './VersionsSidebar.scss';
 
 const MAX_VERSIONS = DEFAULT_FETCH_END;
 
 type Props = {
     error?: MessageDescriptor,
+    errorUpsell: boolean,
     fileId: string,
     isLoading: boolean,
+    onUpgradeCTAClick: openUpgradePlanModal,
```

Comment:
Can we make this more generic, like `onUpgrade: () => void` so the element isn't aware of how this callback will be used?